packages/pharos/src/components/toggle-button-group/pharos-toggle-button.ts

```diff
@@ -0,0 +1,100 @@
+import { property, state } from 'lit-element';
+import type { CSSResultArray, PropertyValues } from 'lit-element';
+import { toggleButtonStyles } from './pharos-toggle-button.css';
+import { designTokens } from '../../styles/variables.css';
+import { customElement } from '../../utils/decorators';
+import { PharosButton } from '../button/pharos-button';
+import type { ButtonType, LinkTarget, IconName } from '../button/pharos-button';
+
+export type { ButtonType, LinkTarget, IconName };
+
+/**
+ * Pharos toggle button component.
+ *
+ * @element pharos-toggle-button
+ *
+ * @slot - Contains the content of the button (the default slot).
+ *
+ */
+@customElement('pharos-toggle-button')
+export class PharosToggleButton extends PharosButton {
+  /**
+   * Indicates that the button is currently toggled on and cannot be pressed or focused by the user.
+   * @attr selected
+   */
+  @property({ type: Boolean, reflect: true })
+  public selected = false;
+
+  @state()
+  private _first = false;
+
+  @state()
+  private _last = false;
```

Comment:
Is the first/last stuff being used somewhere. I see we have the config here and set the props in the button group but I can't tell what we do with it.

Reply:
ahh nvm, see that we have it in the css.

Reply:
We set it in `changedProperties`: https://github.com/ithaka/pharos/pull/25/files#diff-80a667883e25da1a73b5f6425a87be6834136a194fc09a6b7db246278e14392fR65-R71

That sets the attributes that we use in the CSS.